ideascube/mediacenter/views.py

```diff
@@ -16,6 +16,9 @@
 from ideascube.decorators import staff_member_required
 from ideascube.mixins import FilterableViewMixin, OrderableViewMixin
 
+# Do not import directly Catalog as we want to mock it for test.
```

Comment:
Yes, but I would have to mock `ideascube.mediacenter.views.Catalog`. (And everywhere else the Catalog class is imported).

Tests should no be dependent from where the class is imported. They should mock the class only where it is defined (ideascube.serveradmin.catalog)